lib/node_modules/@stdlib/blas/base/srot/src/srot.c

```diff
@@ -31,40 +32,8 @@
 * @param s        sine of the angle of rotation
```

Comment:
@aman-095 Would you mind fixing the descriptions for `X` and `Y` here and elsewhere to be consistent with the README? Namely, to `first input array` and `second input array`.